pkg/yang/identity.go

```diff
@@ -95,13 +98,6 @@ func (mod *Module) findIdentityBase(baseStr string) (*resolvedIdentity, []error)
 			errs = append(errs, fmt.Errorf("%s: can't resolve the local base %s as %s", source, baseStr, keyName))
 		}
 	default:
-		// The identity we are looking for is prefix:basename.  If
```

Comment:
Just to clarify here -- is there any guarantee that this function gets an argument with `prefix:base`? 

I guess this statement can never be executed because if that were the case then `basePrefix` would be `""` such that this is unnecessary?

Reply:
Thanks for looking at this, I must have read this code wrong the first time since this if block can only be called when the argument is `prefix:base`, and not anything else (notice the switch statement). This directly contradicts my PR description :facepalm: I'll revert back this snippet.